introduction_to_amazon_algorithms/jumpstart_question_answering/README.md

```diff
@@ -0,0 +1,2 @@
+### SageMaker JumpStart Question Answering Training & Deployment
+This notebook `Amazon_JumpStart_Question_Answering.ipynb` for  Extractive Question Answering task demonstrates (a) how to use a pre-trained model available in JumpStart for inference, and (b) how to fin-etune a pre-trained Transformer model on a custom dataset using JumpStart transfer learning algorithm, and how to use the fine-tuned model for inference.
```

Comment:
Typos: `fin-etune` -> `fine-tune`, extra space in `for  Extractive`

Reply:
fixed.